docs/source/models.rst

```diff
@@ -784,6 +784,60 @@ method to load MLflow Models with the ``prophet`` model flavor in native prophet
 
 For more information, see :py:mod:`mlflow.prophet`.
 
+Pmdarima (``pmdarima``)
+^^^^^^^^^^^^^^^^^^^^^^^
+The ``pmdarima`` model flavor enables logging of `pmdarima models <http://alkaline-ml.com/pmdarima/>`_ in MLflow
+format via the :py:func:`mlflow.pmdarima.save_model()` and :py:func:`mlflow.pmdarima.log_model()` methods.
+These methods also add the ``python_function`` flavor to the MLflow Models that they produce, allowing the
+model to be interpreted as generic Python functions for inference via :py:func:`mlflow.pyfunc.load_model()`.
+This loaded PyFunc model can only be scored with a DataFrame input.
+You can also use the :py:func:`mlflow.pmdarima.load_model()` method to load MLflow Models with the ``pmdarima``
+model flavor in native pmdarima formats.
+
+.. note::
+    When predicting a ``pmdarima`` flavor, the ``predict`` method argument ``return_conf_int`` controls the
```

Comment:
Nit: `return_conf_int` isn't a method argument, right? It's an optional column in the input pandas dataframe.

Reply:
very good point. Changed this note to make it far more clear what is going on with this configuration (and that by omitting the column in the configuration argument DataFrame, the default will be False for calculating CI values.